Remove remaining references to classic models (#386)
DeepLcom/api-docs: docs/xml-and-html-handling/tag-handling-v2.mdx

**Q: Does v2 work with all DeepL API features?**

> A: Tag handling v2 works with all standard API features including glossaries and formality settings. It is compatible with all `model_type` values.

Reviewer: ```suggestion
A: Tag handling v2 works with all standard API features and `model_type`s.
```
